### PR TITLE
add JSON converter support for User task localization

### PR DESCRIPTION
**Pull request by mo-nick**
user task localization support in JSON converter
#### Check List:
* Unit tests: YES 
* Documentation: NA

### CODE REVIEW

File: modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/BaseBpmnJsonConverter.java [hunk 1]
```diff
@@ -17,42 +17,8 @@
...
-import org.flowable.bpmn.model.FormProperty;
-import org.flowable.bpmn.model.FormValue;
-import org.flowable.bpmn.model.Gateway;
-import org.flowable.bpmn.model.GraphicInfo;
-import org.flowable.bpmn.model.Lane;
-import org.flowable.bpmn.model.MapExceptionEntry;
-import org.flowable.bpmn.model.MessageEventDefinition;
-import org.flowable.bpmn.model.MessageFlow;
-import org.flowable.bpmn.model.MultiInstanceLoopCharacteristics;
+import org.flowable.bpmn.model.*;
```
Comment (dbmalkovsky) [inline modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/BaseBpmnJsonConverter.java]: Project standards are not to use wildcard imports; please update files as needed; thanks.

Comment (mo-nick) [reply modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/BaseBpmnJsonConverter.java]: done!

File: modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/BaseBpmnJsonConverter.java [hunk 2]
```diff
@@ -557,6 +527,63 @@ protected void addEventProperties(Event event, ObjectNode propertiesNode) {
...
+                    String documentation = null;
+                    List<ExtensionElement> documentationElements = localizationElement.getChildElements().get("documentation");
+                    if (documentationElements != null) {
+                        for (ExtensionElement documentationElement : documentationElements) {
+                            documentation = StringUtils.trimToNull(documentationElement.getElementText());
+                            break;
+                        }
+                    }
+                    if (name != null && !name.trim().isEmpty()) {
+                        nameNode.put(locale, name);
```
Comment (dbmalkovsky) [inline modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/BaseBpmnJsonConverter.java]: If you are expecting leading/trailing spaces (the use of trim()); wouldn't you want to store the trimmed value too?

Comment (mo-nick) [reply modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/BaseBpmnJsonConverter.java]: yes, actually I simplified the if condition using StringUtils as well.

Comment (dbmalkovsky) [reply modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/BaseBpmnJsonConverter.java]: thanks

**Comment by tijsrademakers** [comment]
Closing because Modeler is not part of Flowable 7 codebase.

**Pull request closed without merging**